app/utils.py

```diff
@@ -898,6 +901,21 @@ def __init__(self, request, response, env):
         if self.auth and not self.auth.is_valid:
             self.auth = None
 
+        # check admin instead of "login: admin" in app.yaml
```

Comment:
Please be clearer, e.g., "Shows a custom error page here when the user is not an admin instead of..."